wpiutil/src/main/java/edu/wpi/first/util/sendable2/Sendable.java

```diff
@@ -0,0 +1,68 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package edu.wpi.first.util.sendable2;
+
+/**
+ * Interface for sendable serialization.
+ *
+ * <p>Idiomatically, classes that support sendable serialization should provide a static final
+ * member named "sendable" that provides an instance of an implementation of this interface.
+ *
+ * @param <T> object type
+ */
+public interface Sendable<T> {
+  /**
+   * Gets the Class object for the stored value.
+   *
+   * @return Class
+   */
+  Class<T> getTypeClass();
+
+  /**
+   * Gets the dashboard type string.
+   *
+   * @return dashboard type string
+   */
+  String getTypeString();
+
+  /**
+   * Gets the sendable set for an object instance.
+   *
+   * @return sendable set
+   */
+  SendableSet getSendableSet(T obj);
+
+  /**
+   * Gets whether the Sendable should be treated as an actuator.
+   *
+   * @param value true if actuator, false if not
+   */
+  default boolean isActuator() {
+    return false;
+  }
+
+  /**
+   * Sets an object instance to a safe state. This is called
+   * when entering and exiting Live Window mode.
+   */
+  default void setSafeState() {
+  }
```

Comment:
These two are very LW-specific, I suggest removing them.

Reply:
If we're removing LiveWindow, yes, that makes sense.

Reply:
Even if we want to keep the functionality, I suggest we reimplement it in a way that is less coupled (`isActuator` could be done via a generic metadata API).